sdk/datafactory/Azure.ResourceManager.DataFactory/src/Generated/Models/PostgreSqlV2LinkedService.cs

```diff
@@ -18,18 +18,21 @@ public partial class PostgreSqlV2LinkedService : DataFactoryLinkedServicePropert
         /// <param name="server"> Server name for connection. Type: string. </param>
         /// <param name="username"> Username for authentication. Type: string. </param>
         /// <param name="database"> Database name for connection. Type: string. </param>
+        /// <param name="authenticationType"> The authentication type to use. Type: string. </param>
         /// <param name="sslMode"> SSL mode for connection. Type: integer. 0: disable, 1:allow, 2: prefer, 3: require, 4: verify-ca, 5: verify-full. Type: integer. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="server"/>, <paramref name="username"/>, <paramref name="database"/> or <paramref name="sslMode"/> is null. </exception>
-        public PostgreSqlV2LinkedService(DataFactoryElement<string> server, DataFactoryElement<string> username, DataFactoryElement<string> database, DataFactoryElement<int> sslMode)
+        /// <exception cref="ArgumentNullException"> <paramref name="server"/>, <paramref name="username"/>, <paramref name="database"/>, <paramref name="authenticationType"/> or <paramref name="sslMode"/> is null. </exception>
+        public PostgreSqlV2LinkedService(DataFactoryElement<string> server, DataFactoryElement<string> username, DataFactoryElement<string> database, DataFactoryElement<string> authenticationType, DataFactoryElement<int> sslMode)
```

Comment:
This is a breaking change. Need to write custom codes to make the API compatible. For examples, see: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/compute/Azure.ResourceManager.Compute/src/Customize